components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -3153,7 +3194,7 @@ private void setGroupDisplayName(Group oldGroup, Group newGroup)
         String userStoreDomainFromSP = getUserStoreDomainFromSP();
 
         String oldGroupDomain = IdentityUtil.extractDomainFromName(oldGroup.getDisplayName());
-        if(userStoreDomainFromSP != null && !userStoreDomainFromSP.equalsIgnoreCase(oldGroupDomain)) {
+        if (userStoreDomainFromSP != null && !userStoreDomainFromSP.equalsIgnoreCase(oldGroupDomain)) {
```

Comment:
let's use `StringUtils.isNotBlank` instead.

Reply:
In here only a formatting issue is fixed. `userStoreDomainFromSP != null` is used in other places also.